.github/scripts/compare-artifact-package-versions.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "Artifact Registry Provider = $ARTIFACT_REGISTRY_PROVIDER"
+
+# TODO: add support for AWS Code artifact
+if [ $ARTIFACT_REGISTRY_PROVIDER = 'npm' ]; then
+
+  echo "Checking npm registry for current version"
+  NPM_PACKAGE_VERSION=`npm show @terraware/web-components version`
+  echo "Got npm package version $NPM_PACKAGE_VERSION"
+
+  echo "Checking version in git package.json"
+  PACKAGE_JSON_VERSION=`node -e "process.stdout.write(require(__dirname + '../../dist/package.json').version)"`
```

Comment:
Do we update this version manually? If we make a change in a component for example?

Reply:
For now yes. We can evolve this over time.